.github/workflows/contract-monitor.yml

```diff
@@ -0,0 +1,56 @@
+name: Monitor Stellar Contract
+
+on:
+  schedule:
+    - cron: '0 8 * * 1' #every Monday at 8:00 AM
+
+jobs:
+  monitor-contract:
+    runs-on: ubuntu-latest
+    container:
+      image: node:20
+    permissions:
+      contents: write
+    env:
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          apt-get update
+          apt-get install -y curl unzip gnupg libdbus-1-3
+
+      - name: Install Stellar CLI
+        run: |
+          mkdir -p /tmp/stellar-cli
+          curl -L https://github.com/stellar/stellar-cli/releases/download/v22.5.0/stellar-cli-22.5.0-x86_64-unknown-linux-gnu.tar.gz \
```

Comment:
I am not a fan. Why not install from cargo?

Also we will want to add some caching.